Public/Src/IDE/VsCode/client/package.json

```diff
@@ -20,7 +20,8 @@
   "main": "./out/src/extension",
   "activationEvents": [
     "onLanguage:DScript",
-    "onCommand:DScript.reloadWorkspace"
+    "onCommand:DScript.reloadWorkspace",
+    "workspaceContains:config.dsc"
```

Comment:
maybe 'config.bc' as well

Reply:
nah, I'll update osgtools one day :)

Reply:
Sadly it spread out to other places as well (like the CB repo). But that's fine, feel free to ignore my comment.